### PR TITLE
Portfolio Mobile menu

### PR DESCRIPTION
**Pull request by telanet**
### In this milestone we implemented the following:
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,381 +17,242 @@
...
-    <li><a href="#"> <img src="images/Icon-Angelist.png" alt="Angelist"></a> </li>
-    <li><a href="#"><img src="images/Icon-Twitter.png" alt="twitter"> </a> </li>
-    <li><a href="#"><img src="images/Icon-Medium.png" alt="Medium"> </a> </li>
-   </ul>
+    <ul class="social-media-icons ul-list horizontal-list">
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-github"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-linkedin"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-angellist"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-twitter"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fab fa-medium-m"></a></li>
```
Comment (ShoiraTa) [inline index.html:60]: - [ ] OPTIONAL: Please decrease the size of icons, after introduced changes, they become huge: 
Screenshot 2022-04-04 at 20 39 55

File: script.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const navBar = document.querySelector('.navbar');
+const menuBtn = document.querySelector('.menu-btn');
+const cancelBtn = document.querySelector('.cancel-btn');
+
+menuBtn.onclick = function () {
+  navBar.classList.add('active');
+  menuBtn.style.opacity = '0';
+  menuBtn.style.pointerEvents = 'none';
+  body.style.overflow = 'hidden';
+};
```
Comment (ShoiraTa) [inline script.js:11]: - [ ] After user clicks some menu item, `overflow = hidden;` property stays there and does not allow users to scroll up and down. Kindly remove overflow = 'hidden'; property when mobile menu is not open ⭐ 

https://user-images.githubusercontent.com/77038610/161591751-80e31341-de25-49a6-bfdc-f8f35bdeca29.mp4

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @telanet ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @telanet,


## Wow, you did it 🎉
Brilliant
## Thank you for correcting the icons, and removing the `overflow:hidden` property, thus correcting the flow of your menu, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**